pages/knn-classification.py

```diff
@@ -5,17 +5,19 @@
 
 dash.register_page(
     __name__,
-    description="Visualize scikit-learn's k-Nearest Neighbors (kNN) classification",
+    description="Visualize scikit-learn's k-Nearest Neighbors (kNN) classification. It has a top-bottom layout and a regular-callback.",
 )
 
 filename = __name__.split("pages.")[1]
 
 
 notes = """
 
-#### Plotly Documentation:  
+#### Dash Components in App:
+- [Slider](https://dash.plotly.com/dash-core-components/slider)
 
-- [Visualize scikit-learn's k-Nearest Neighbors (kNN) classification](https://plotly.com/python/knn-classification/)
+#### Plotly Documentation:  
+- [Scatter Plots](https://plotly.com/python/line-and-scatter/)
 
```

Comment:
This app also has a GO.contour plot. Can you add this link under the scatter plot link:
https://plotly.com/python/contour-plots/

Reply:
Done